src/components/Card.js

```diff
@@ -1,11 +1,25 @@
 import '../styles/card.css';
-import noneImg from '../assets/noneImg.png';
+import noneImg from '../assets/noneImg.svg';
+import starIcon from '../assets/starIcon.svg';
+import meatballsIcon from '../assets/meatballsIcon.svg';
+
 function Card({ id, time, imgUrl = noneImg, title, description, date, url }) {
   return (
-    <a className="card" key={id} href={url} target="_blank">
-      <img className="card__img" src={imgUrl} alt={title}></img>
+    <a className="card" key={id} href={url} target="_blank" rel="noreferrer">
+      <div className="card__img">
+        <img className="card__img--background" src={imgUrl} alt={title}></img>
+        <img
+          className="card__img--meatsball-icon"
+          src={starIcon}
+          alt={'Favorites'}
```

Comment:
사소하지만, 바로 문자열을 붙이는걸 추천드립니다
```suggestion
          alt="Favorites"
```